### DISCUSSION TITLE
use code-server written markdown documents cannot be previewed

### DISCUSSION THREAD

**Discussion by gglss** [coder/code-server#5222] [Q&A]
environment：
docker-centos7
use https
![image](https://user-images.githubusercontent.com/81339751/169955877-e7a7d5dc-b4aa-4307-b7dd-0c2bfaae355e.png)
Click preview screen is empty and an error is reported

**Comment by jsjoeio** [comment]
It seems like it your service workers aren't working. Are there any other errors in the browser console?

**Reply by gglss** [reply]
![image](https://user-images.githubusercontent.com/81339751/170157137-ee507d7e-a853-4458-8cf5-48cf5ef7560f.png)
That's all the console got
The certificate I use is this
![image](https://user-images.githubusercontent.com/81339751/170157397-e8fe8496-0f3f-4213-9a6d-b021b0fe4cdf.png)
Is it certificate?

**Reply by jsjoeio** [reply]
I think so :( Seems like a limitation of the browser: https://github.com/coder/code-server/issues/3410#issuecomment-1135479029

**Reply by gglss** [reply]
okay,thanks
